docs/layouts.rst

```diff
@@ -301,8 +301,30 @@ These ones live under module ``crispy_forms.bootstrap``.
 .. image:: images/field_disabled.png
    :align: center
 
-.. _`input_size`:
+- **Modal**: ``Modal`` displays it's fields inside a bootstrap modal that can be customized usign kwargs upon initialization. Only supports Bootstrap v3 or higher::
 
+    Layout(
+        BootstrapModal(
+            # email.help_text was set during the initalization of the django form field
+            Field('email', placeholder="Email", wrapper_class="mb-0"),
+            Button(
```

Comment:
Gotcha, I left out how to close the modal with buttons because the bootstrap docs explain how to do that
[via attributes](https://getbootstrap.com/docs/4.0/components/modal/#via-data-attributes) and [via javascript](https://getbootstrap.com/docs/4.0/components/modal/#via-javascript). I'll add these links to the docs.